test/unit_test/static/chrono_test.cpp

```diff
@@ -67,12 +73,12 @@ static_assert(!std::constructible_from<time_point<std::chrono::system_clock, si:
 static_assert(!std::convertible_to<sys_seconds, time_point<std::chrono::system_clock, si::day, sys_seconds::rep>>);
 
 // construction - different rep type (integral to a floating-point)
-static_assert(std::constructible_from<si::time<si::second>, std::chrono::seconds>);
-static_assert(!std::convertible_to<std::chrono::seconds, si::time<si::second>>);
-static_assert(std::constructible_from<si::time<si::second>, std::chrono::hours>);
-static_assert(!std::convertible_to<std::chrono::hours, si::time<si::second>>);
-static_assert(std::constructible_from<si::time<si::hour>, std::chrono::seconds>);
-static_assert(!std::convertible_to<std::chrono::seconds, si::time<si::hour>>);
+static_assert(std::constructible_from<si::time<si::second, double>, std::chrono::seconds>);
+static_assert(!std::convertible_to<std::chrono::seconds, si::time<si::second, double>>);
+static_assert(std::constructible_from<si::time<si::second, double>, std::chrono::hours>);
+static_assert(!std::convertible_to<std::chrono::hours, si::time<si::second, double>>);
+static_assert(std::constructible_from<si::time<si::hour, double>, std::chrono::seconds>);
+static_assert(!std::convertible_to<std::chrono::seconds, si::time<si::hour, double>>);
```

Comment:
Defaults for template arguments do not work with modules?

Reply:
That's one of the issues listed in #7: https://github.com/llvm/llvm-project/issues/54340.

Reply:
This is not required after the realization stated at https://github.com/mpusz/units/pull/350#discussion_r1126285956.